src/cpu/cpu.cpp

```diff
@@ -2425,25 +2433,12 @@ class CPU final : public Module_base {
 	}
 };
 
-static CPU * test;
```

Comment:
"test"... Why not "apricot"? Or "truckDriver"? 🤣

Reply:
better than a single underscore! `CPU* _`